start scrub of glossary (#741)
MicrosoftDocs/quantum-docs-pr: articles/concepts/Glossary.md

|Term|Definition|
|-------------|----------|

> |<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an operation. For operations that implement a unitary operator, the adjoint is the inverse of the operation. Adjoints are indicated by dagger symbol, for example the adjoint of vector $v$ is $v^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|

Reviewer: ```suggestion
|<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an operation. For operations that implement a unitary operator, the adjoint is the inverse of the operation. Adjoints are indicated by the dagger symbol, for example, the adjoint of an operator $A$ is $A^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|
```
Although the adjoint of a vector is mathematically well defined, I think it is better to use operators in the example, since they are more relevant in the context of quantum computing.

Author: updated

> |<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an operation. For operations that implement a unitary operator, the adjoint is the inverse of the operation. Adjoints are indicated by dagger symbol, for example the adjoint of vector $v$ is $v^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|

Author: updated

> |<a id="bell-state"></a>Bell state|A maximally entangled quantum state of two qubits. Also known as an [EPR pair](xref:microsoft.quantum.glossary#epr)|

Author: updated

|<a id="callable"></a>Callable|In the Q# language, operations and functions are collectively known as *callables*. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|

> |<a id="clifford"></a>Clifford group|The set of operations that occupy the octants of the Bloch sphere and effect permutations of the Pauli operators. These include: `X`, `Y`, `Z`, `H` and `S`|

Author: updated

|<a id="controlled"></a>Controlled|A quantum operation that takes one or more qubits as enablers for the target operation. For more information, see [Controlled](xref:microsoft.quantum.language.type-model#controlled).|
|<a id="dirac"></a>Dirac Notation|A shorthand representation of quantum states, for example $$\frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}=H\ket{0} = \ket{+}$$. For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|

> |<a id="eigenvalue"></a>Eigenvalue| The factor by which the magnitude of an [eigenvector](xref:microsoft.quantum.glossary#eigenvector) is changed by a given transformation. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|

Reviewer: ```suggestion
|<a id="eigenvalue"></a>Eigenvalue| The factor by which the magnitude of an [eigenvector](xref:microsoft.quantum.glossary#eigenvector) of a given transformation is changed by the application of the transformation. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
```

> |<a id="eigenvalue"></a>Eigenvalue| The factor by which the magnitude of an [eigenvector](xref:microsoft.quantum.glossary#eigenvector) is changed by a given transformation. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|

Reviewer 2: The term "magnitude" often refers to a nonnegative real number, but an eigenvalue can in general be a complex number of any argument (phase).

> |<a id="eigenvector"></a>Eigenvector|A vector whose direction is unchanged by a given transformation and whose magnitude is changed by a factor corresponding to that vector's [eigenvalue](xref:microsoft.quantum.glossary#eigenvalue). For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|

Reviewer 2: Similarly here wrt "magnitude."

> |<a id="epr"></a>EPR pair|A maximally entangled quantum state of two qubits. Also known as a [Bell state](xref:microsoft.quantum.glossary#bell-state)|

Reviewer 2: Similar comment here as with the Bell state comment.

|<a id="evolution"></a>Evolution|How a quantum state changes over time. For more information, see [Matrix exponentials](xref:microsoft.quantum.concepts.matrix-advanced#matrix-exponentials).|

> |<a id="function"></a>Function|A purely classical (non-quantum) routine in the Q# language. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|

Reviewer 2: Notably, not all purely classical routines are functions! Random number sampling, machine environment queries (e.g.: those in Microsoft.Quantum.Environment) and other nondeterminstic classical logic is also represented by operations.

|<a id="global-phase"></a>Global phase|When two states are identical up to a multiple of a complex number $e^{i\phi}$, they are said to differ up to a global phase. Unlike local phases, global phases cannot be observed through any measurement. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|

> |<a id="measurment"></a>Measurement|Obtaining a classical bit from a qubit (or set of qubits). FOr more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit#measuring-a-qubit).|

Reviewer 2: ```suggestion
|<a id="measurment"></a>Measurement|Obtaining a classical bit from a qubit (or set of qubits). For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit#measuring-a-qubit).|
```

|<a id="mutable"></a>Mutable|A variable whose value may be changed after it is created.|

> |<a id="namespace"></a>Namespace|A label for a collection of related names (typically operations, functions, and types). For instance the namespace [Microsoft.Quantum.Preparation](xref:microsoft.quantum.preparation) labels all of the symbols defined in the standard library that help with preparing initial states.|

Reviewer 2: ```suggestion
|<a id="namespace"></a>Namespace|A label for a collection of related names (i.e.: operations, functions, and user-defined types). For instance the namespace [Microsoft.Quantum.Preparation](xref:microsoft.quantum.preparation) labels all of the symbols defined in the standard library that help with preparing initial states.|
```

|<a id="operation"></a>Operation|The basic unit of quantum execution in Q#. It is roughly equivalent to a function in C, C++ or Python, or a static method in C# or Java. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
|<a id="operator-application"></a>Operator application|Performing a quantum operation. This typically applies a unitary matrix to the current quantum   state vector.|
|<a id="oracle"></a>Oracle|A subroutine that provides data-dependent information to a quantum algorithm at runtime. Typically, the goal is to provide a superposition of outputs corresponding to inputs that are in superposition. For more information, see [Oracles](xref:microsoft.quantum.libraries.data-structures#oracles).|

> |<a id="partial-app"></a>Partial application|Calling a function or operation without all the required parameters. The returns a new callable that only needs the missing parameters supplied during a future application. For more information, see [Partial application](xref:microsoft.quantum.libraries.data-structures#partial-application).|

Reviewer 2: ```suggestion
|<a id="partial-app"></a>Partial application|Calling a function or operation without all the required inputs. The returns a new callable that only needs the missing parameters supplied during a future application. For more information, see [Partial application](xref:microsoft.quantum.libraries.data-structures#partial-application).|
```
A brief example may also be useful here.

> |<a id="pauli"></a>Pauli operators|The `X`, `Y` and `Z` quantum gates. For more information, see [Single-qubit operations](xref:microsoft.quantum.concepts.qubit#single-qubit-operations).|

Reviewer 2: At a least, I'd suggest "operations" instead of "gates," as a "gate" can be a colloquialism for either an operation or an operator, depending on context. That said, the original definition here was likely wrong in that it conflated 𝑋, `PauliX` and `Microsoft.Quantum.Intrinsic.X` with each other.
```suggestion
|<a id="pauli"></a>Pauli operators|The `X`, `Y` and `Z` quantum operations. For more information, see [Single-qubit operations](xref:microsoft.quantum.concepts.qubit#single-qubit-operations).|
```

> |<a id="prelude"></a>Prelude|The set of primitive and classical operations and functions defined by each individual target machine, rather than at the Q# level For more information, see [The prelude](xref:microsoft.quantum.libraries.standard.prelude).|

Reviewer: ```suggestion
|<a id="prelude"></a>Prelude|The set of primitive and classical operations and functions defined by each individual target machine, rather than at the Q# level. For more information, see [The prelude](xref:microsoft.quantum.libraries.standard.prelude).|
```

> |<a id="prelude"></a>Prelude|The set of primitive and classical operations and functions defined by each individual target machine, rather than at the Q# level For more information, see [The prelude](xref:microsoft.quantum.libraries.standard.prelude).|

Reviewer 2: This entire entry is probably outdated, and can be safely removed; it used to matter more before we unified a bunch of library content with 0.6.
```suggestion
```

> |<a id="circuit"></a>Quantum circuit|A diagrammatical representation of a program for a quantum computer. FOr more information, see [Quantum circuits](xref:microsoft.quantum.concepts.circuits).|

Reviewer: ```suggestion
|<a id="circuit"></a>Quantum circuit|A diagrammatical representation of a program for a quantum computer. For more information, see [Quantum circuits](xref:microsoft.quantum.concepts.circuits).|
```

> |<a id="circuit"></a>Quantum circuit|A diagrammatical representation of a program for a quantum computer. FOr more information, see [Quantum circuits](xref:microsoft.quantum.concepts.circuits).|

Reviewer 2: It's probably worth noting that circuits represent a very restricted view of quantum programs (i.e.: all operations unrolled and inlined into intrinsics, and only extremely limited classical feedforward allowed), such that Q# programs in general do not necessarily have representations as circuits.
```suggestion
|<a id="circuit"></a>Quantum circuit|A diagrammatical representation of a program for a quantum computer. For more information, see [Quantum circuits](xref:microsoft.quantum.concepts.circuits).|
```

> |<a id="quantum-state"></a>Quantum state|A representation of the qubits in a system, usually denoted as a complex column vector called the *quantum state vector*. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|

Reviewer 2: I'm not quite sure what the original definition meant by "representation of the qubits in a system." Effectively, a state is what a simulator has to record internally in order to simulate how a qubit responds to operations.
```suggestion
|<a id="quantum-state"></a>Quantum state|A representation of the qubits in a system, usually denoted as a complex column vector called the *quantum state vector*. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|
```

|<a id="qubit"></a>Qubit|A basic unit of quantum storage. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|

> |<a id="rus"></a>Repeat-until-success|A quantum algorithm that probabilistically succeeds. Upon failure, the routine will re-try until successful (or a limit has been reached). For more information, see [Repeat ntil Success (RUS)](xref:microsoft.quantum.techniques.qubits#measurements)|

Reviewer 2: ```suggestion
|<a id="rus"></a>Repeat-until-success|A quantum algorithm that probabilistically succeeds. Upon failure, the routine will retry until successful (or a limit has been reached). For more information, see [Repeat Until Success (RUS)](xref:microsoft.quantum.techniques.qubits#measurements)|
```

> |<a id="standard"></a>Standard|Operations and functions defined in Q# that build on the logic defined in the [prelude](xref:microsoft.quantum.glossary#prelude). The standard library implementation is agnostic with respect to target machines. For more information, see [Standard libraries](xref:microsoft.quantum.libraries.standard.intro).|

Reviewer 2: This may need modified slightly if you decide to adopt my above suggestion about removing the "prelude" entry.

> |<a id="target"></a>Target machine|A compilation target that lowers an abstract quantum program towards hardware or simulation. This typically include re-writes for many purposes including gate replacement, encoding for error correction, geometric layout and others. For more information, see [Quantum simulators and host applications](xref:microsoft.quantum.machines).|

Reviewer 2: I'd suggest soliciting @bettinaheim's input for this one.

|<a id="tuple"></a>Tuple|Comma separated types grouped together via parenthesis. For more information, see [Tuple types](xref:microsoft.quantum.language.type-model#tuple-types).|

> |<a id="unitary"></a>Unitary operator|An operator whose inverse is equal to its adjoints.|

Reviewer 2: ```suggestion
|<a id="unitary"></a>Unitary operator|An operator whose inverse is equal to its adjoint (i.e.: $UU^{\dagger} = \id$).|
```

|Term|Definition|
|-------------|----------|

> |<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an operation. For operations that implement a unitary operator, the adjoint is the inverse of the operation and is indicated by a dagger symbol. For example, if the operation `U` represents the unitary operator $U$, then `Adjoint U` represents $U^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|

Author: Added this and other inline glossary links

|<a id="ancilla"></a>Ancilla|A [qubit](xref:microsoft.quantum.glossary#qubit) that serves as temporary memory for a quantum computer and is allocated and de-allocated as needed.  For more information, see [Multiple qubits](xref:microsoft.quantum.concepts.multiple-qubits).|
|<a id="bell-state"></a>Bell state|One of four specific maximally [entangled](xref:microsoft.quantum.glossary#entanglement) [quantum states](xref:microsoft.quantum.glossary#quantum-state) of two qubits. The four states are defined $\ket{\beta_{ij}} = (\mathbb{I} \otimes X^iZ^j) (\ket{00} + \ket{11}) / \sqrt{2}$. A Bell state is also known as an [EPR pair](xref:microsoft.quantum.glossary#epr).|

> |<a id="bloch"></a>Bloch sphere|A graphical representation of a single-[qubit](xref:microsoft.quantum.glossary#qubit) [quantum state](xref:microsoft.quantum.glossary#quantum-state) as a three-dimensional real-valued vector. For more information, see  [Visualizing Qubits and Transformations using the Bloch Sphere](xref:microsoft.quantum.concepts.qubit#visualizing-qubits-and-transformations-using-the-bloch-sphere).|

Reviewer 3: I'm not so sure how we define it as real-valued. It can contain phase information of the qubit's coefficients.

Reviewer 4: I assume this is just referring to the Bloch vector ([described here](https://en.wikipedia.org/wiki/Bloch_sphere)), which is a real-valued vector. But I agree, maybe here this could say "as a point in a three-dimensional unit sphere" instead of "as a three-dimensional real-valued vector"? The [cross-reference](https://docs.microsoft.com/en-us/quantum/concepts/the-qubit#visualizing-qubits-and-transformations-using-the-bloch-sphere) discusses the real-valued Bloch vector anyway.

|<a id="callable"></a>Callable|An [operation](xref:microsoft.quantum.glossary#operation) or [function](xref:microsoft.quantum.glossary#function) in the Q# language. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
|<a id="clifford"></a>Clifford group|The set of operations that occupy the octants of the [Bloch sphere](xref:microsoft.quantum.glossary#bloch) and effect permutations of the [Pauli operators](xref:microsoft.quantum.glossary#pauli). These include the operations [$X$](xref:microsoft.quantum.intrinsic.x), [$Y$](xref:microsoft.quantum.intrinsic.y), [$Z$](xref:microsoft.quantum.intrinsic.z), [$H$](xref:microsoft.quantum.intrinsic.h) and [$S$](xref:microsoft.quantum.intrinsic.s).|
|<a id="controlled"></a>Controlled|A quantum [operation](xref:microsoft.quantum.glossary#operation) that takes one or more [qubits](xref:microsoft.quantum.glossary#qubit) as enablers for the target operation. For more information, see [Controlled](xref:microsoft.quantum.language.type-model#controlled).|

> |<a id="dirac"></a>Dirac Notation|A symbolic shorthand that simplifies the representation of [quantum states](xref:microsoft.quantum.glossary#quantum-state), also called *bra-ket* notation.  The *bra* portion represents a row vector, for example  $\bra{A} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix}$ and the *ket* portion represents a column vector, $\ket{B} = \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix}$.  Bras and kets can be combined, for example $\braket{A \| B} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix} \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix} = A{_1}B{_1} + A{_2}B{_2}$, or can represent the same quantum state in different ways, for example, $\ket{+} = H \ket{0} = \frac{1}{\sqrt{2}} \left(\ket{0} + \ket{1} \right) = \frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}$. For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|

Reviewer 3: I'm having a bit of trouble with the phrasing in the explanation of Dirac Notation, especially that "can represent the same quantum state in different ways" is not a scientific description. The example given is more to show that a quantum state can be decomposed into other quantum states, rather than explaining Diract notation. It may be better just to remove it or say it can be used to represent any state in a quantum system. Also the example of them being combined is only the inner product. There's also the outer product and generally speaking they act as operators. So perhaps say they can represent individual states or be combined to act as operators. (And there is an "operator application" in the glossary.)

|<a id="entanglement"></a>Entanglement|Quantum particles, such as qubits, can be connected or *entangled* such that actions performed on one immediately duplicate on the other. Entanglement is essential to measuring the state of a qubit.  For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
|<a id="epr"></a>EPR pair|One of four specific maximally entangled quantum states of two qubits. The four states are defined $\ket{\beta_{ij}} = (\mathbb{1} \otimes X^iZ^j) (\ket{00} + \ket{11}) / \sqrt{2}$. An EPR pair is also known as a [Bell state](xref:microsoft.quantum.glossary#bell-state)|
|<a id="evolution"></a>Evolution|How a quantum state changes over time. For more information, see [Matrix exponentials](xref:microsoft.quantum.concepts.matrix-advanced#matrix-exponentials).|
|<a id="function"></a>Function|A type of subroutine in the Q# language that is purely classical (non-quantum). While functions are used within quantum algorithms, they may not act on qubits or call operations. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|

> |<a id="hadamard"></a>Hadamard|The Hadamard operation (also referred to as the Hadamard gate or transform) acts on a single qubit and puts it in superposition. In Q#, it is the pre-defined [$H$](xref:microsoft.quantum.intrinsic.h) operation.|

Reviewer 2: Technically, the `H` operation only produces an even superposition of |0⟩ and |1⟩ when acting on a qubit initially in the |0⟩ state. As a side note, this line conflates the `Microsoft.Quantum.Intrinsic.H` operation with the unitary matrix 𝐻 = (¹₁ ¹₋₁) / √2 used to simulate the `H` operation.

> |<a id="gate"></a>Gate|A legacy term for a quantum operation, based on the concept of classical logic gates.|

Reviewer 5: Should we say that a gate is a matrix operation (because the gates and operators mentioned in the rest of the glossary are used somewhat interchangeably)?

Reviewer 2: That's part of why I recommend against using "gate" as much as possible. The term is equally often applied to unitary matrices (e.g.: 𝐻) and to intrinsic Q# operations (e.g.: `H`), introducing confusion. Put differently, the definition that you give is correct, @natke, but is one of two mutually contradictory definitions in common use at the moment.

> |<a id="entanglement"></a>Entanglement|Quantum particles, such as [qubits](xref:microsoft.quantum.glossary#qubit), can be connected or *entangled* such that actions performed on one immediately duplicate on the other. Entanglement is essential to [measuring](xref:microsoft.quantum.glossary#measurement) the [state](xref:microsoft.quantum.glossary#quantum-state) of a qubit.  For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|

Reviewer 3: This is not true. "immediately duplicate" implies faster-than-light information transfer, which is prohibited. Can we change it to "...entangled such that they cannot be described independently from each other. Their measurement results are correlated even when they are separated infinitely far away."

|<a id="global-phase"></a>Global phase|When two states are identical up to a multiple of a complex number $e^{i\phi}$, they are said to differ up to a global phase. Unlike local phases, global phases cannot be observed through any measurement. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|

> |<a id="immutable"></a>Immutable|A variable whose value cannot be changed. An immutable is created using the `let` keyword. To declare variables that *can* be changed, use the [mutable](xref:microsoft.quantum.glossary#immutable) keyword to declare and the `set` keyword to modify the value. |

Reviewer 5: Suggest:
`An immutable variable in Q# is created ...`

|<a id="partial-app"></a>Partial application|Calling a function or operation without all the required inputs. This returns a new callable that only needs the missing parameters (indicated by an underscore) to be supplied during a future application. For example, given the function `MyFunc(x : int, y : int) : int {return x + y;}` you can partially apply it to a new function `let NewFunc = MyFunc(_, 3)`. You can then call the new function at a later time with the missing parameter `NewFunc(2)` which returns the value *5*.  For more information, see [Partial application](xref:microsoft.quantum.language.expressions#partial-application).|
|<a id="pauli"></a>Pauli operators|A set of three 2 x 2 unitary matrices known as the `X`, `Y` and `Z` quantum operations. The identity matrix, $I$, is often included in the set as well.  $I = \begin{bmatrix} 1 & 0 \\\\ 0 & 1 \end{bmatrix}$, $X = \begin{bmatrix} 0 & 1 \\\\ 1 & 0 \end{bmatrix}$, $Y = \begin{bmatrix} 0 & -i \\\\ i & 0 \end{bmatrix}$, $Z = \begin{bmatrix} 1 & 0 \\\\ 0 & -1 \end{bmatrix}$.   For more information, see [Single-qubit operations](xref:microsoft.quantum.concepts.qubit#single-qubit-operations).|

> |<a id="circuit"></a>Quantum circuit diagram|A method to graphically represent the sequence of operations for simple quantum programs. For more information, see [Quantum circuits](xref:microsoft.quantum.concepts.circuits).|

Reviewer 5: Should we say that a circuit is a network of gates (to extend the classical analogy). I know we are trying to deprecate the gate/circuit terminology but if we include it here, we should be clear about what it is)

|<a id="quantum-libraries"></a>Quantum libraries|Collections of operations, functions and user-defined types for creating Q# programs. The [Standard library](xref:microsoft.quantum.libraries.standard.intro) is installed by default. Other libraries available are the [Chemistry library](xref:microsoft.quantum.chemistry.concepts.intro), the [Numerics library](xref:microsoft.quantum.numerics.intro) and the [Machine learning library](xref:microsoft.quantum.machine-learning.concepts.intro).|

> |<a id="quantum-state"></a>Quantum state|In quantum computing, the information about a qubit that can be physically measured. The quantum simulator uses this information to simulate how a qubit responds to operations. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|

Reviewer 4: qubit or set of qubits?

> |<a id="qubit"></a>Qubit|A basic unit of quantum storage. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|

Reviewer 5: quantum storage -> quantum information?

|<a id="rus"></a>Repeat-until-success|A quantum algorithm that probabilistically succeeds. Upon failure, the routine will retry until successful (or a limit has been reached). For more information, see [Repeat Until Success (RUS)](xref:microsoft.quantum.techniques.qubits#measurements)|
|<a id="standard"></a>Standard libraries|Operations, functions and user-defined types that are installed along with the Q# compiler during installation. The standard library implementation is agnostic with respect to target machines. For more information, see [Standard libraries](xref:microsoft.quantum.libraries.standard.intro).|

> |<a id="superposition"></a>Superposition|The concept in quantum computing that a qubit can hold two states at once, for example $\ket{\0}$ and $\ket{\1}$, until it is measured.  For more information, see [What is quantum computing](xref:microsoft.quantum.overview.what).|

Reviewer 5: can hold two states -> can hold infinite combinations of two states

|<a id="target"></a>Target machine|A compilation target that lowers an abstract quantum program towards hardware or simulation. This typically include re-writes for many purposes including gate replacement, encoding for error correction, geometric layout and others. For more information, see [Quantum simulators and host applications](xref:microsoft.quantum.machines).|

> |<a id="teleporation"></a>Teleportation|A method for moving data, or the [quantum state](xref:microsoft.quantum.glossary#quantum-state), of one qubit to another using [entanglement](xref:microsoft.quantum.glossary#entanglement) and [measurement](xref:microsoft.quantum.glossary#measurement).  For more information, see [Quantum circuits](xref:microsoft.quantum.concepts.circuits) and [Putting it all together](xref:microsoft.quantum.techniques.puttingittogether).|

Reviewer 4: id="teleportation"

> |<a id="quantum-state"></a>Quantum state|In quantum computing, the information about a [qubit](xref:microsoft.quantum.glossary#qubit) that can be physically [measured](xref:microsoft.quantum.glossary#measurement). The quantum simulator uses this information to simulate how a qubit responds to operations. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|

Reviewer 6: This might seem to imply that 1) "quantum states" specifically refer to just qubits, and 2) that a "quantum state" is fully defined by the immediately measurable data (if that were the case, I think either there would be no reason for Q# to exist in the first place, or we would already be in the year 2100---too late in the day to think about which).
Perhaps the following?
```suggestion
|<a id="quantum-state"></a>Quantum state|The precise state of a quantum system, from which [measurement](xref:microsoft.quantum.glossary#measurement) probabilities can be extracted, as well as how it will evolve subject to any operation. The quantum simulator uses this information to simulate how qubits respond to operations. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|
```

|<a id="callable"></a>Callable|An [operation](xref:microsoft.quantum.glossary#operation) or [function](xref:microsoft.quantum.glossary#function) in the Q# language. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
|<a id="clifford"></a>Clifford group|The set of operations that occupy the octants of the [Bloch sphere](xref:microsoft.quantum.glossary#bloch) and effect permutations of the [Pauli operators](xref:microsoft.quantum.glossary#pauli). These include the operations [$X$](xref:microsoft.quantum.intrinsic.x), [$Y$](xref:microsoft.quantum.intrinsic.y), [$Z$](xref:microsoft.quantum.intrinsic.z), [$H$](xref:microsoft.quantum.intrinsic.h) and [$S$](xref:microsoft.quantum.intrinsic.s).|
|<a id="controlled"></a>Controlled|A quantum [operation](xref:microsoft.quantum.glossary#operation) that takes one or more [qubits](xref:microsoft.quantum.glossary#qubit) as enablers for the target operation. For more information, see [Controlled](xref:microsoft.quantum.language.type-model#controlled).|

> |<a id="dirac"></a>Dirac Notation|A symbolic shorthand that simplifies the representation of [quantum states](xref:microsoft.quantum.glossary#quantum-state), also called *bra-ket* notation.  The *bra* portion represents a row vector, for example  $\bra{A} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix}$ and the *ket* portion represents a column vector, $\ket{B} = \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix}$.  Bras and kets can be combined, for example $\braket{A \| B} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix} \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix} = A{_1}B{_1} + A{_2}B{_2}$, or can represent the same quantum state in different ways, for example, $\ket{+} = H \ket{0} = \frac{1}{\sqrt{2}} \left(\ket{0} + \ket{1} \right) = \frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}$. For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|

Reviewer 6: I agree with Kitty---easiest just to cut the elaboration here (it's already quite lengthy compared to other entries) and let the hairs be split in the actual article.

```suggestion
|<a id="dirac"></a>Dirac Notation|A symbolic shorthand that simplifies the representation of [quantum states](xref:microsoft.quantum.glossary#quantum-state), also called *bra-ket* notation.  The *bra* portion represents a row vector, for example  $\bra{A} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix}$ and the *ket* portion represents a column vector, $\ket{B} = \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix}$.  For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|
```
